snare/tests/test_add_scheme.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+import os
+import yarl
+import shutil
+from snare.cloner import Cloner
+from snare.utils.page_path_generator import generate_unique_path
+
+
+class TestCloner(unittest.TestCase):
+    def setUp(self):
+        self.url = 'http://example.com/'
+        self.main_page_path = generate_unique_path()
```

Comment:
Is this required?

Reply:
Yes, CI was failing without this.

Reply:
Sorry about that, will keep that in mind. But it failed because the path `/opt/snare/pages/` not existed while running this test.